### DISCUSSION TITLE
ISSUE THAT COST ME HOURS

### DISCUSSION THREAD

**Discussion by simonenotarangelo** [Klocman/Bulk-Crap-Uninstaller#653] [General]
So, i installed a program that i placed on my main folder that i use to save my photos, my project and all my personal stuff.
I forgot that i located the program there, so like always i uninstalled it from BCU. The problem is here!! BCU thoght that all my folder was part of the program, but the folder's tree was like this:  *MYFOLDER*>OldPhotos, MyProject, JDownloader. So i tried to uninstall JDownloader that was only placed here. BCU just decided to uninstall all the MYFOLDER, and it even said Good when it looked for the remaining file(that were in the folder. PLS SOLVE, ADD MORE CHECKS, I LOST EVERYTHING.

**Comment by Klocman** [comment]
Check your recycle bin. Also why did you accept if you saw that the path shown in BCU was wrong?
